xml/System.IO.Pipelines/PipeWriter.xml

```diff
@@ -349,10 +358,10 @@ You must request a new buffer after calling <xref:System.IO.Pipelines.PipeWriter
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>Writes <paramref name="source" /> to the pipe and makes data accessible to <see cref="T:System.IO.Pipelines.PipeReader" /></summary>
-        <returns>To be added.</returns>
+        <param name="source">The read-only byte span to write.</param>
+        <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see cref="T:System.Threading.CancellationToken.None"/>.</param>
+        <summary>Writes <paramref name="source" /> to the pipe and makes data accessible to <see cref="T:System.IO.Pipelines.PipeReader" />.</summary>
+        <returns>A task that represents the asynchronous write operation, and wraps the flush asynchronous operation.</returns>
```

Comment:
Is this correct? Does it both write and flush?

Reply:
That seems to be the case. If you look at the code, it writes, then asynchronously flushes, and returns the flush task:

https://github.com/dotnet/corefx/blame/bb0516a4f8d0c4d84eff2ba2a3ad0a4764ece9d2/src/System.IO.Pipelines/src/System/IO/Pipelines/PipeWriter.cs#L98